src/kite.js

```diff
@@ -50,6 +50,8 @@ export const Kite = {
   },
 
   _activate() {
+    this.globalState.setKeysForSync(["kite.showWelcomeNotificationOnStartup"]);
```

Comment:
perhaps we want to only run this code if the API supports it, since this API is new